src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/ComparisonServiceTest.kt

```diff
@@ -0,0 +1,136 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.service
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mockito
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.security.core.context.SecurityContextImpl
+import org.springframework.security.oauth2.jwt.Jwt
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.config.AuthAwareAuthenticationToken
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.entity.Comparison
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.external.ComparisonInput
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.repository.ComparisonPersonRepository
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.repository.ComparisonRepository
+import java.time.LocalDateTime
+import java.util.UUID
+
+@ExtendWith(MockitoExtension::class)
+class ComparisonServiceTest : IntegrationTestBase() {
+
+  private val prisonService = mock<PrisonService>()
+
+  @Autowired
+  lateinit var comparisonService: ComparisonService
```

Comment:
Do you mean `@Autowired` or do you mean `@InjectMocks`?

Reply:
Done :)